app/views/categories/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<body>
+  <main>
+    <h2>Products by Category</h2>
+
+  <section class= "product-list">
+  <h4>Annual</h4>  
+    <% @product.categories each do |annual| %>   
+    <h4><%= product.name.capitalize %></h4>
+      <ul>
+        <li>Description: <%= product.description.capitalize %></li>
+        <li>Price: $<%= product.price %></li>
+        <li>Stock: <%= product.stock %> units</li>
+        <li>Available: <%= product.available %></li>
```

Comment:
'Available' is a boolean, which is the field where the merchant decides if the product is viewable, yes? I'm thinking that we need a custom method that will sort out the products that are marked as unavailable and possibly where stock is zero. No urgent need to change this now and you've perhaps you've already considered this for a future iteration.

Reply:
I think the product status (available) is attached to the merchant "deleting it" and not stock. This is just a view for the merchant... I think for stock we only need to avoid that the user is going to add a product with zero stock to their order.